tests/baselines/reference/divergentAccessorsTypes1.js

```diff
@@ -0,0 +1,107 @@
+//// [divergentAccessorsTypes1.ts]
+class Test1 {
+    get foo(): string { return "" }
+    set foo(s: string | number) {
+        let a = s as string;
+        let b = s as number;
+    }
+
+    get bar(): string | number { return "" }
+    set bar(s: string | number | boolean) {
+    }
+}
+
+interface Test2 {
+    get foo(): string;
+    set foo(s: string | number);
+
+    get bar(): string | number;
+    set bar(s: string | number | boolean);
+}
+
+type Test3 = {
+    get foo(): string;
+    set foo(s: string | number);
+
+    get bar(): string | number;
+    set bar(s: string | number | boolean);
+};
+
+{
+    const t = new Test1();
+    t.foo = 32;
+    let m: string = t.foo;
+
+    // See how CFA interacts with out-of-type writes
```

Comment:
42 is compatible with `get bar(): string | number`, though. Is this comment correct?

Later: oh, the comment refers to `t.bar = false`, and is actually comparing in-type vs out-of-type writes.
```suggestion
    // See how CFA interacts with in-type vs out-of-type writes
```